pipelines/cdp/scripts/00_set_variables.sh

```diff
@@ -0,0 +1,22 @@
+# This file is generated by the Terraform code of this Solution Guide.
```

Comment:
This script should be generated by the Terraform code and not included in the repo. Please remove it, as it contains some hardcoded values.